src/Microsoft.AspNetCore.Authorization/AuthorizationOptions.cs

```diff
@@ -11,7 +11,10 @@ namespace Microsoft.AspNetCore.Authorization
     /// </summary>
     public class AuthorizationOptions
     {
-        private IDictionary<string, AuthorizationPolicy> PolicyMap { get; } = new Dictionary<string, AuthorizationPolicy>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Yeah this is a bit leaky since we aren't adding it to the actual interface (and I don't think we necessarily want to either, since enumerating all legal policies names isn't something that's required today).  But it seems fine for the concrete Options to expose access to its dictionary.  

How about we just expose it as a `IReadOnlyDictionary<string, AuthorizationPolicy>`?